specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -377,7 +377,12 @@ Unit and integration tests will be introduced by following the official Airflow
 
 The communication between VDK Provider and Control Service will be secured through the
 use of HTTPS and OAuth 2.0. The most commonly used OAuth 2.0 flows will be supported (
-e.g. Client Credentials Flow, Refresh Token Flow, etc.).
+e.g. Client Credentials Flow, Refresh Token Flow, etc.) by using `vdk-control-cli`'s authentication
+functionality. This would make `vdk-control-cli` a hard dependency of the Provider, though
+it is possible for the functionality to be moved as a separate library.
```

Comment:
```suggestion
it is possible for the functionality to be moved as a separate library.
```

Let's be more definite. Please make a recommendation should it be moved or not? 
I personally think yes. But we can discuss pros and cons

Reply:
Good point. Refactored, thanks